core/cc/log.cpp

```diff
@@ -129,32 +129,39 @@ void Logger::vlogf(unsigned level, const char* src_file, unsigned src_line,
   }
 #else   // TARGET_OS != GAPID_OS_ANDROID
 
+  // Note that we use "GAPID" as the logcat tag, rather than mInstance.mSystem,
+  // in order to easily filter all GAPID-related logcat. The mInstance.mSystem
+  // is still present in the message.
+
   std::stringstream ss;
+  if (mInstance.mSystem != nullptr && strlen(mInstance.mSystem) > 0) {
+    ss << mInstance.mSystem << " ";
+  }
   ss << "[" << src_file << ":" << src_line << "] " << format;
 
   switch (level) {
     case LOG_LEVEL_FATAL:
-      __android_log_assert(nullptr, mInstance.mSystem, ss.str().c_str(), args);
+      // There is no va_list version of __android_log_assert(), hence the
+      // message pre-formatting in this case.
+      char buf[2048];
+      vsnprintf(buf, sizeof(buf), format, args);
```

Comment:
src_file and src_line will be absent from the message?

Reply:
Indeed! I pushed a fix. Thanks!